sandboxed_containers/troubleshooting-sandboxed-containers.adoc

```diff
@@ -0,0 +1,25 @@
+[id="troubleshooting-sandboxed-containers"]
+= Collecting {sandboxed-containers-first} data for Red Hat Support
+include::modules/common-attributes.adoc[]
+:context: troubleshooting-sandboxed-containers
+
+toc::[]
+
+[role="_abstract"]
+
+When opening a support case, it is helpful to provide debugging
+information about your cluster to Red Hat Support.
+
+The `must-gather` tool enables you to collect diagnostic information about your
+{product-title} cluster, including virtual machines and other data related to
+{sandboxed-containers-first}.
+
+For prompt support, supply diagnostic information for both {product-title}
+and {sandboxed-containers-first}.
+
+include::modules/about-must-gather.adoc[leveloffset=+1]
+
+include::modules/sandboxed-containers-collecting-data.adoc[leveloffset=+1]
+
+== Additional resources
```

Comment:
You need to add an ID statement to this ==Additional Resources, unless you want to use it as .Additional Resources.

Reply:
Will add ASAP